### ISSUE TITLE
The man pages should respect the value of --with-rcsd and --with-config

### ISSUE THREAD

**Issue by RoastVeg** [troglobit/finit#182]
As per the title, setting up autotools and creating `finit.8.in` et al should make this always correct.

**Comment by troglobit** [comment]
Yup, that would be a useful improvement.  I rushed out the man pages in 4.1 using defaults for the above.
